spec/ten_thousand_feet/api/assignment_spec.rb

```diff
@@ -3,22 +3,21 @@
 module TenThousandFeet
   module API
     describe Assignments do
-      
-      let!(:client)  { TenThousandFeet.new({ auth: $AUTH }) }
+      let!(:client)  { TenThousandFeet.new auth: ENV['auth'] }
```

Comment:
In our projects we tend to use all caps for environment variables. This pattern is also followed in dotenv's README.md file. https://github.com/bkeepers/dotenv/blob/master/README.md

In light of this, will you please change `'auth'` to `'AUTH'`?